client/containers/DashboardOverview/overviewRows/labels.tsx

```diff
@@ -78,3 +76,52 @@ export const getPubReleasedStateLabel = (pub: Pub) => {
 		icon: 'lock' as const,
 	};
 };
+
+export const getSubmissionStatusLabel = (
+	submissionStatus: Partial<SubmissionStatus>,
+): IconLabelPair => {
+	if (submissionStatus === 'accepted') {
+		return {
+			label: 'Accepted',
+			icon: 'symbol-square' as const,
```

Comment:
Does this work if you remove `as const`? I kinda think it should but I'm not sure.